Civi/Api4/Service/Spec/FieldSpec.php

```diff
@@ -15,6 +15,7 @@
 /**
  * Contains APIv4 field metadata
  */
+#[AllowDynamicProperties]
 class FieldSpec {
```

Comment:
For most of the changes, +1

On this part (marking `FieldSpec` as dynamic), I'm not sure. It looks to me (on a skim/grep) like the class is pretty structured (ie traits/getters/setters). Are there some examples where it's dynamic?

Reply:
@totten I was finding that most of the api_v4_SyntaxConformanceTests were triggering this but it is possible I suppose the wrong Spec class was being used